docker/usr/local/toolchain/generate_g4dnxlarge_toolchain.py

```diff
@@ -0,0 +1,32 @@
+#! /usr/bin/env python3
```

Comment:
I would love a comment at the top of this explaining what this target is for. Future years will thank us